integration_tests/test_basic.py

```diff
@@ -679,7 +679,7 @@ def test_tx_inclusion(cronos, max_gas_wanted):
     """
     modify_command_in_supervisor_config(
         cronos.base_dir / "tasks.ini",
-        lambda cmd: f"{cmd} --evm.max-tx-gas-wanted {max_gas_wanted}",
+        lambda _: f"cronosd start --home . --trace --evm.max-tx-gas-wanted {max_gas_wanted}",
```

Comment:
cmd append with existing like `cronosd start --home . --trace --evm.max-tx-gas-wanted 80000000 --evm.max-tx-gas-wanted 40000000`, not sure it's a bug since just last one take effect @yihuang